admin/release.py

```diff
@@ -879,7 +875,7 @@ def initialize_release(version, path, top_level):
         os.environ["LDFLAGS"] = '-L{}/lib" CFLAGS="-I{}/include'.format(
             brew_openssl, brew_openssl)
     check_call(
-        ["pip install -e .[dev]"], shell=True,
+        ["pip", "install", "-e" ".[dev]"],
```

Comment:
missing comma after -e I think? Might still work cause single letter args sometimes parse all characters after the first as the arg parameter. If that's the case and it is intentional, prefer to delete the space between the strings.